eventstore/bbolt/go.mod

```diff
@@ -1,12 +1,14 @@
 module github.com/hallgren/eventsourcing/eventstore/bbolt
 
-go 1.17
+go 1.21
+
+toolchain go1.22.5
```

Comment:
Should the `toolchain` feature be used in a package where I have no idea how it's compiled? Just because I have go version 1.22.5 installed others could have 1.21.10. Would that be okey as it's below 1.22?

Reply:
Remove it for now.